src/gf-platformid

```diff
@@ -41,19 +41,21 @@ coreos_gf_run_mount "${tmp_dest}"
 # * grub config
 # * BLS config (for subsequent config regeneration)
 # First, the grub config.
-if [ "$(coreos_gf exists '/boot/efi')" == 'true' ]; then
-    grubcfg_path=$(coreos_gf glob-expand /boot/efi/EFI/*/grub.cfg)
-else
-    grubcfg_path=/boot/loader/grub.cfg
+if [ "$arch" -ne "x86_64" ]; then
```

Comment:
This is called by `buildextend-metal` too though. Needs a `|| [ "$platformid" == "metal" ]`, right?

Reply:
Oh yup, good catch. Fixed.